runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -2681,6 +2681,80 @@ J9::ARM64::TreeEvaluator::VMnewEvaluator(TR::Node *node, TR::CodeGenerator *cg)
       {
       genInitArrayHeader(node, cg, clazz, resultReg, classReg, lengthReg, zeroReg, tempReg1, isBatchClearTLHEnabled, tlhHasNotBeenCleared);
 
+#ifdef TR_TARGET_64BIT
```

Comment:
You don't need this `#ifdef` line, because AArch64 is always a 64-bit environment.

Reply:
ah right right, good point. I have removed it in my latest commit. Thank you for the catch!